src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -3248,6 +3244,7 @@
     <EmbeddedResource Include="NHSpecificTest\NH1291AnonExample\Mappings.hbm.xml" />
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="NHSpecificTest\NH3985\Mappings.hbm.xml" />
```

Comment:
Thanks. 

These tags become harmful when you try to merge branches as they provide false anchors for the merge tools.